packages/terra-switch/src/Switch.module.scss

```diff
@@ -0,0 +1,108 @@
+:local {
+  .switch-wrapper {
+    display: inline-block;
+    -webkit-tap-highlight-color: transparent;
+    width: auto;
+  }
+
+  .switch-container {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .label-container {
+    display: flex;
+    flex-direction: column;
+    margin-right: 1.45rem;
+    margin-top: 0.28571rem;
+  }
+
+  .label-text {
+    color: var(--terra-switch-label-color, #1c1f21);
```

Comment:
shouldn't we set ellipsis here ? if the text length exceeds ?

Reply:
No, because It should Wrap text whenever length exceeds  as per the [requirements](https://app.zeplin.io/project/59b1bfeae515657fa8268300/screen/5b8eb1ffc19e7c1fda41d32b) (Responsive Variation)